jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -118,16 +119,27 @@ pipeline {
                         def CUDA_NAME = sh(returnStdout: true,
                                 script: '. jenkins/version-def.sh>&2 && echo -n $CUDA_CLASSIFIER | sed "s/-/./g"')
 
-                        IMAGE_TAG = "dev-ubuntu16-${CUDA_NAME}"
-                        CACHE_IMAGE_NAME = "${ARTIFACTORY_NAME}/sw-spark-docker/plugin:${IMAGE_TAG}"
+                        // check if pre-merge dockerfile modified
+                        def dockerfileModified = sh(returnStdout: true,
+                                script: 'BASE=$(git --no-pager log --oneline -1 | awk \'{ print $NF }\'); ' +
+                                        'git --no-pager diff --name-only HEAD $(git merge-base HEAD $BASE) ' +
+                                        '-- jenkins/Dockerfile-blossom.ubuntu16 || true')
```

Comment:
It would be nice to store the name of the CI Dockerfile in a variable at the top of this script and use the variable throughout so if we change the file path later it later it's easy to update.

Reply:
sure, let me update it